source/tools/kokkos-connector/CMakeLists.txt

```diff
@@ -115,6 +119,7 @@ configure_library(kp_timemory_filter)
 #
 if(TIMEMORY_KOKKOS_BUILD_SAMPLE)
     add_executable(sample sample/sample.cpp)
+    set_target_properties(${PROJECT_NAME} PROPERTIES DEBUG_POSTFIX ${CMAKE_DEBUG_POSTFIX})
```

Comment:
This can be removed if `CMAKE_DEBUG_POSTFIX` is cached

Reply:
No, I do not think so. Unlike ``add_library`` ``add_executable`` by itself does not use the debug postfix when naming the executable generated by the debug build. As far as I know the only way to get the debug postfix into the generated executable name is to have a ``set_target_properties`` for every ``add_executable``. (I have a [stackoverflow answer](https://stackoverflow.com/a/29023980/3154588), which is a little old, but I just checked and [\<CONFIG\>_POSTFIX](https://cmake.org/cmake/help/latest/prop_tgt/CONFIG_POSTFIX.html#config-postfix) still says the same thing.)

Reply:
Ah gotcha. But why do the executables need to have the suffix if all the libs, etc. have them?

Reply:
So that when you install a debug executable it does not overwrite the install of the release executable, which is almost always what you want when you install release and debug targets to the same directory. A debug postfix for executable targets is not needed if you install debug targets to a separate directory, which timemory is not doing. (Nor, in my experience, do most typical cmake projects.)